web/src/styles/_text.scss

```diff
@@ -1,57 +1,53 @@
 .heading {
-	@apply font-[400] text-secondary;
+	@apply font-semibold text-primary mb-[32px];
+}
+
+.h1 {
+	@extend .heading;
+
+	@apply text-xl;
 
 	&::before {
 		content: ">_ ";
 	}
+}
 
-	@screen sm {
-		@apply mb-5;
-	}
-
-	@screen md {
-		@apply mb-6;
-	}
+.h2 {
+	@extend .heading;
 
-	@screen lg {
-		@apply mb-7;
-	}
+	@apply font-fira text-lg;
 
-	@screen xl {
-		@apply mb-8;
+	&::before {
+		content: "> ";
 	}
 }
 
-.h1 {
-	@extend .heading;
-	font-size: clamp(3rem, 5vw, 5rem);
+p.paragraph {
+	@apply font-fira font-normal text-base text-white;
 }
 
-.h2 {
-	@extend .heading;
-	font-size: clamp(2rem, 4vw, 4rem);
-}
+a.link {
+	&:link {
+		@apply font-normal font-semibold text-base text-white underline;
+	}
 
-.h3 {
-	@extend .heading;
-	font-size: clamp(1.75rem, 3.5vw, 3rem);
-}
+	&:visited {
+		@apply text-gray-1;
+	}
 
-.h4 {
-	@extend .heading;
-	font-size: clamp(1.5rem, 3vw, 2.5rem);
-}
+	&:hover {
+		@apply text-primary;
+	}
 
-.h5 {
-	@extend .heading;
-	font-size: clamp(1.5rem, 2.5vw, 1.75rem);
-}
+	&:focus {
+		@apply text-red-600;
+	}
 
-.h6 {
-	@extend .heading;
-	font-size: clamp(1.5rem, 2vw, 1.25rem);
+	&:active {
+		@apply text-primary;
+	}
 }
 
-ul.list li {
+.text- ul.list li {
 	list-style-type: "> ";
```

Comment:
Does the character „>“ look nice as list style element in this font?

Reply:
I checked and I had to adjust the html of the list to get some space between the marker and the list-item content.
By default, the markers overflow the parent ul-element on the left side. The css-property `list-style-position: inside` did not work because it added a line break between the marker and the list item text.

Pls. have a look again.

Reply:
can send a screenshot?

Reply:
looks good in the screen cast 👍🏻